newrelic/resource_newrelic_workload.go

```diff
@@ -0,0 +1,327 @@
+package newrelic
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+	"github.com/newrelic/newrelic-client-go/pkg/workloads"
+)
+
+func resourceNewRelicWorkload() *schema.Resource {
+	userReference := &schema.Schema{
```

Comment:
I'd argue that this provides a little bit of confusion because it's been pulled out this way. I could see pulling this schema out if it had more nested values but I'd suggest leaving it in the main schema otherwise. I think it's only used in an attribute that we can remove but you can take this info and apply it going forward.

Reply:
👍